osc-server/src/main/java/org/osc/core/broker/util/network/NetworkSettingsApi.java

```diff
@@ -148,4 +88,46 @@ public NetworkSettingsDto getNetworkSettings() {
         return dns;
     }
 
-}
+    private String getNetMask(String ipAddr) {
+        String[] ipAddrParts = ipAddr.split("\\.");
+        String mask = "";
+        int octet = Integer.parseInt(ipAddrParts[0]);
+        if (octet <= 127) {
+            mask = "255.0.0.0";
+        } else if (octet >= 128 && octet <= 191) {
```

Comment:
netmask is not limited to the 5 mask in this code. 32 netmasks are possible. can we figure out by parsing the command line just like the other network settings?

see org.osc.core.broker.util.NetworkUtil.validNetMasks